modules/ROOT/pages/ign-storage.adoc

```diff
@@ -114,6 +114,43 @@ storage:
 <4> We use the previously defined partitions as devices in a RAID1 md array.
 <5> The resulting md array is used to create an EXT4 file system.
 
+.Example for adding a /var partition to the primary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+storage:
+  disks:
+  - device: /dev/vda <1>
+    wipe_table: false <2>
+    partitions:
+    - number: 5 <3>
+      size_mib: 0
+      start_mib: 0
+      label: var <4>
+  filesystems:
+    - path: /var
+      device: /dev/disk/by-partlabel/var
+      format: xfs
+systemd:
+  units:
+    - name: var.mount <5>
+      enabled: true
+      contents: |
+        [Unit]
+        Before=local-fs.target
+        [Mount]
+        Where=/var
+        What=/dev/disk/by-partlabel/var
+        [Install]
+        WantedBy=local-fs.target
+----
+<1> The name of the primary block device. In virtio-based setups, this is likely `/dev/vda`. Elsewhere, it's likely `/dev/sda`.
```

Comment:
Rather than footnotes, let's just use comments within the YAML doc.  (Should probably fix the other examples to do the same.)

Reply:
I think we should do this, though I think it's more important to be consistent and I don't want to tackle the other examples right now.